arcgis-ios-sdk-samples/Maps/Take screenshot/MapViewScreenshotViewController.swift

```diff
@@ -83,12 +83,12 @@ class MapViewScreenshotViewController: UIViewController {
     }
     
     //imitate the white flash screen when the user taps on the screenshot button
-    private func imitateFlash(image:UIImage) {
+    fileprivate func imitateFlash(_ image:UIImage) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage